src/Evolution/Systems/ScalarTensor/System.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/VariablesTag.hpp"
+#include "Evolution/Systems/CurvedScalarWave/System.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/System.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+/*!
+ * \ingroup EvolutionSystemsGroup
+ * \brief Items related to evolving the first-order scalar tensor system.
```

Comment:
Please add some docs here :) Add the equations and/or link to the GH and CSW systems.

Reply:
Added an expanded description below.